liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -95,6 +95,12 @@
      */
     @Parameter(property = "serverName", defaultValue = "defaultServer")
     protected String serverName = null;
+
+    /**
+     * Location of customized configuration file server.xml in devmode
+     */
+    @Parameter(alias="configFile", property = "serverXmlFile")
+    protected File serverXmlFile;
```

Comment:
So by moving this to BasicSupport, you have made this a `common parameter` instead of a `common server parameter`...which doesn't really make sense...as it is specific to specific server. I'm trying to figure out what implications that might have and whether we will need to update those two documentation pages to reflect this. Most of the parameters in BasicSupport apply to the Liberty install and not to a particular server in that install. Although I see `serverName` right above here which applies to a server as well.

Reply:
This field could be located here but still logically be a `common server parameter` and documented as such. I don't think it will have an impact on the other mojos because Maven manages it. It is still worthwhile looking for unexpected effects.